node/cmd/guardiand/node.go

```diff
@@ -836,33 +795,6 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger.Fatal("Infura is known to send incorrect blocks - please use your own nodes")
 	}
 
-	ethContractAddr := eth_common.HexToAddress(*ethContract)
-	bscContractAddr := eth_common.HexToAddress(*bscContract)
-	polygonContractAddr := eth_common.HexToAddress(*polygonContract)
-	avalancheContractAddr := eth_common.HexToAddress(*avalancheContract)
-	oasisContractAddr := eth_common.HexToAddress(*oasisContract)
-	auroraContractAddr := eth_common.HexToAddress(*auroraContract)
-	fantomContractAddr := eth_common.HexToAddress(*fantomContract)
-	karuraContractAddr := eth_common.HexToAddress(*karuraContract)
-	acalaContractAddr := eth_common.HexToAddress(*acalaContract)
-	klaytnContractAddr := eth_common.HexToAddress(*klaytnContract)
-	celoContractAddr := eth_common.HexToAddress(*celoContract)
-	moonbeamContractAddr := eth_common.HexToAddress(*moonbeamContract)
-	neonContractAddr := eth_common.HexToAddress(*neonContract)
-	arbitrumContractAddr := eth_common.HexToAddress(*arbitrumContract)
-	optimismContractAddr := eth_common.HexToAddress(*optimismContract)
-	baseContractAddr := eth_common.HexToAddress(*baseContract)
-	solAddress, err := solana_types.PublicKeyFromBase58(*solanaContract)
-	sepoliaContractAddr := eth_common.HexToAddress(*sepoliaContract)
-	if err != nil {
-		logger.Fatal("invalid Solana contract address", zap.Error(err))
```

Comment:
This error message gets lost in the changes.  Not sure if it is extremely important.

Reply:
This check is in https://github.com/wormhole-foundation/wormhole/blob/2fdb18e29ffdc572ab5a4f222c4d8e05a9aef652/node/pkg/watchers/solana/config.go#L48 and would also generate a logger.Fatal in https://github.com/wormhole-foundation/wormhole/blob/2fdb18e29ffdc572ab5a4f222c4d8e05a9aef652/node/pkg/node/node.go#L250.